samza-rest/src/main/java/org/apache/samza/monitor/SamzaMonitorService.java

```diff
@@ -43,16 +51,14 @@
   private static final Logger LOGGER = LoggerFactory.getLogger(SamzaMonitorService.class);
   private static final SecureRandom RANDOM = new SecureRandom();
 
-  private final SchedulingProvider scheduler;
```

Comment:
If this is the only place this interface was being used, let's delete the interface + impl.

Reply:
Deleted the unused interface and it's impl